### ISSUE TITLE
[Expo 46] duplicate expo-constants installed

### ISSUE THREAD

**Issue by chunghe** [expo/expo#20459]
### Summary

Steps:
1. `expo init -t tabs@sdk-46`
2. check yarn.lock, two versions(13.2.4, 14.0.2) of expo-constants installed. 

```
expo-constants@~13.2.4:
  version "13.2.4"
  resolved "https://registry.yarnpkg.com/expo-constants/-/expo-constants-13.2.4.tgz#eab4a553f074b2c60ad7a158d3b82e3484a94606"
  integrity sha512-Zobau8EuTk2GgafwkfGnWM6CmSLB7X8qnQXVuXe0nd3v92hfQUmRWGhJwH88uxXj3LrfqctM6PaJ8taG1vxfBw==
  dependencies:
    "@expo/config" "~7.0.0"
    uuid "^3.3.2"

expo-constants@~14.0.0:
  version "14.0.2"
  resolved "https://registry.yarnpkg.com/expo-constants/-/expo-constants-14.0.2.tgz#2cb1dec8f41a64c2fc5b4eecaf77d7661cad01cc"
  integrity sha512-wzV3nrzTXTI8yG0tfas3fnqCfKV6YE+1GphEREyVDAShEB6mBInX1b6HgtpHFy2wOtnml+lPVmTCeGtjjLnZhA==
  dependencies:
    "@expo/config" "~7.0.2"
    uuid "^3.3.2"
```

Note:
1. The correct version should be `~13.2.4` (specify in project's package.json), but `expo-constants@~14.0.0` is required for expo-asset
2. expo-asset's another dependency expo-file-system also not aligned of the dependency of Expo 46

```
$ yarn why expo-constants
yarn why v1.22.18
[1/4] 🤔  Why do we have the module "expo-constants"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "expo-constants@13.2.4"
info Has been hoisted to "expo-constants"
info Reasons this module exists
   - Specified in "dependencies"
   - Hoisted from "expo#expo-constants"
```

### What platform(s) does this occur on?

Not Applicable

### Environment

```
$ npx expo-env-info

  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.0.1
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 16.15.0 - ~/.nvm/versions/node/v16.15.0/bin/node
      Yarn: 1.22.18 - ~/.nvm/versions/node/v16.15.0/bin/yarn
      npm: 8.5.5 - ~/.nvm/versions/node/v16.15.0/bin/npm
    Managers:
      CocoaPods: 1.11.3 - /Users/chunghe/.rvm/rubies/ruby-2.7.2/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.1, iOS 16.1, macOS 13.0, tvOS 16.1, watchOS 9.1
    IDEs:
      Android Studio: 4.2 AI-202.7660.26.42.7322048
      Xcode: 14.1/14B47b - /usr/bin/xcodebuild
    npmPackages:
      expo: ~46.0.17 => 46.0.17
      react: 18.0.0 => 18.0.0
      react-dom: 18.0.0 => 18.0.0
      react-native: 0.69.6 => 0.69.6
      react-native-web: ~0.18.7 => 0.18.10
    Expo Workflow: managed
```

### Minimal reproducible example

Steps:
1. `expo init -t tabs@sdk-46`
2. check yarn.lock, two versions of expo-constants installed.

**Comment by Kudo** [comment]
hi there! we've published `expo@46.0.19` and `expo-asset@8.6.3` which should fix the problem. by `expo-cli init -t tabs@sdk-46` should also be resolved. thanks for let us know the problem!

**Comment by chunghe** [comment]
Kudos from Taiwan. I can confirm the issue has fixed.
